### PR TITLE
MEN-4505: Fix terminal PATH issue by defaulting to a login shell on spawn

### PR DESCRIPTION
**Pull request by oleorhagen**
Along with some minor nitpicks and spelling fixes.

### CODE REVIEW

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

File: app/daemon.go [hunk 1]
```diff
@@ -16,12 +16,13 @@ package app
...
 	"fmt"
-	"github.com/mendersoftware/mender-connect/limits/filetransfer"
 	"os/user"
 	"strconv"
 	"sync"
 	"time"
 
+	"github.com/mendersoftware/mender-connect/limits/filetransfer"
+
 	log "github.com/sirupsen/logrus"
```
Comment (merlin-northern) [inline app/daemon.go:26]: maybe one more sort?

File: shell/shell_test.go [hunk 2]
```diff
@@ -45,21 +45,21 @@ func TestMenderShellExecShell(t *testing.T) {
 	}
```
Comment (merlin-northern) [inline shell/shell_test.go:46]: do you think we could add here `[]string{""}` scenario? also, maybe we should test a scenario with bad parameters, what happens then?

Comment (oleorhagen) [reply shell/shell_test.go:46]: Good idea! Fixed!

**Review by merlin-northern — APPROVED** [review]
lgtm! thank you.

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
I cannot see that this should change the coverage in the `dbus_libgio` file ....

**Comment by oleorhagen** [comment]
Cherries:
* 1.1.x: https://github.com/mendersoftware/mender-connect/pull/34
* 1.0.x: incoming

**Pull request merged**